README.md

```diff
@@ -2,5 +2,7 @@
 
 This repository contains the `genesis.json` for Celestia testnets.
 
+If you want to see versions per testnets that are being used, then navigate to this [grafana dashboard](https://celestia.grafana.net/goto/JktIO8zSg?orgId=1)
```

Comment:
[question] is this Grafana dashboard publicly visible? When I navigate to it and login via a non Celestia org account, I observe this error:

<img width="517" alt="Screenshot 2023-09-06 at 2 40 46 PM" src="https://github.com/celestiaorg/networks/assets/3699047/75c80dd5-3888-408d-886a-5cffc63ab21e">

If this dashboard isn't publicly accessible, I think we should maintain a publicly accessible version matrix. In other words, I don't think we should close https://github.com/celestiaorg/networks/issues/159 yet.

btw the dashboard is nice, and I'll keep it bookmarked 👍

Reply:
I've changed the link to a public view, so there is no login needed. 
Thanks for the catch! Should be fine now